### ISSUE TITLE
Add small volume correction option

### ISSUE THREAD

**Issue by nicholst** [NISOx-BDI/SwE-toolbox#21]
Would only be available for parametric, FDR corrections.

**Comment by nicholst** [comment]
Only a priority if could be easily accommodated through code inherited from `spm_getSPM.m`

**Comment by TomMaullin** [comment]
Hi @nicholst,

I have looked into this and found that to do this the following parameters are needed (at least):

```
 - xSwE.VRpv - RESELS per voxel (density) volume
 - xSwE.FWHM - Full width half maximum
 - xSwE.df - degrees of freedom 
```

I guess `df` could be set to the nominal degrees of freedom and we could ask the user to input the `FWHM` they used perhaps? I'm unsure how `VRpv` is calculated however.

**Comment by nicholst** [comment]
Hi @TomMaullin 

Confused... we have a working assumption that SwE doesn't work with RFT.  As noted above, this is *just* for FDR corrections, since spm_P_FDR is quick to run.

Does this help?

**Comment by TomMaullin** [comment]
Ah right apologies, yes. The code from `spm_results_ui.m` calls to `spm_VOI.m` for small volume corrections (see [here](https://github.com/spm/spm12/blob/master/spm_results_ui.m#L604)). This code uses:

- `VRpv` in the call to `spm_list` at line [215](https://github.com/spm/spm12/blob/master/spm_VOI.m#L215)
- `df` in lines [297-299](https://github.com/spm/spm12/blob/master/spm_list.m#L297).

It also uses `FWHM` 3 times (on lines [168](https://github.com/spm/spm12/blob/master/spm_VOI.m#L168), [177](https://github.com/spm/spm12/blob/master/spm_VOI.m#L177) and [199](https://github.com/spm/spm12/blob/master/spm_VOI.m#L199)) but I think these could be removed/worked around if needed.

To adapt the code from SPM as you suggested above these sections of the code would have to be modified/removed. The one thing I am confused about though as we don't currently have full brain clusterwise FDR. Also should the small volume corrections be available and/or parametric in the wild bootstrap display case?

**Comment by TomMaullin** [comment]
This has now been fixed by PR #47
